Formula/emacs-plus.rb

```diff
@@ -233,6 +235,17 @@ class EmacsPlus < Formula
     sha256 "f6cd00bcd37be03d8d83c2e590ab7d259ae37632f5954267240ba9439f799359"
   end
 
+  if build.with? "system-appearance-change"
+    unless build.with? "emacs-27-branch"
```

Comment:
There is a separate 'section'[1] for incompatible options. I find it easier to keep them in one place.

  [1]: https://github.com/d12frosted/homebrew-emacs-plus/pull/206/files#diff-cbfee31de48c4c925ef3e240a0362bb2R156

Reply:
👍🏻 I moved the check to that section.